e2e/utils/form.js

```diff
@@ -161,6 +167,21 @@ class FormSubmitter {
     await this.page.click(datePickerDaySelector);
   }
 
+  async fillDateRangeField(path, value) {
+    const fieldSelector = `[${ID_ATTRIBUTE}="${path}"]`;
+    await this.page.click(fieldSelector);
+    const datePickerStartDateSelector = `[title="${moment(value[0]).format(
+      'MMMM D, YYYY'
+    )}"]`;
+    const datePickerEndDateSelector = `[title="${moment(value[1]).format(
+      'MMMM D, YYYY'
+    )}"]`;
+    await this.page.waitFor(datePickerStartDateSelector);
+    await this.page.click(datePickerStartDateSelector);
+    await this.page.waitFor(datePickerEndDateSelector);
+    await this.page.click(datePickerEndDateSelector);
```

Comment:
```
    const datePickerDaySelector = `[title="${moment(value).format(
      'MMMM D, YYYY'
    )}"]`;
    await this.page.waitFor(datePickerDaySelector);
    await this.page.click(datePickerDaySelector);
```
is in above `fillDateField` is identical to this. So you can extract this to a function, like `selectDateOnActivePicker(date)`